app/assets/javascripts/spotlight/user/browse_group_categories.js

```diff
@@ -25,6 +25,17 @@
       });
     }
 
+    // Destroy the slider instance, as tns will change the dom elements, causing some issues with turbolinks
+    function setupDestroy() {
+      if (Turbolinks.supported) {
```

Comment:
Is it.. possible Turbolinks will be null? Would you be safe just always adding the listener?

Reply:
Thanks for the review/suggestions. I've removed the check, and changed the event. That seems to be working just fine.

Reply:
🎉